### PR TITLE
move subheads out of glossary partials

### PR DESCRIPTION
**Pull request by nancymic2**
fixes: b/270137366
Changes proposed in this pull request:
move subheads out of glossary partials and put them in the glossary index file so heads don't show up in other text/docs when used

https://deploy-preview-5391--developer-chrome-com.netlify.app/docs/privacy-sandbox/glossary/

### CODE REVIEW

**Comment by nancymic2** [comment]
> So, without including the h2 in the partial, we don't have a guarantee of how these will be used in the future. Can you transform the entries to include the glossary item, so it can be used out-of-context? For example:
> 
> ```
> An ad exchange is a platform to automate buying and selling of ad inventory from
> multiple ad networks.
> ```
> 
> Second, can you add a file to the partial directory with some info about these? Call it `_README.md` and include how the entry should be structured (aka "Do not include a heading").
> 
> Once done, will approve

Done. Tests still runing. Here's the readme https://github.com/GoogleChrome/developer.chrome.com/blob/nancy-glossary-partials/site/en/_partials/privacy-sandbox/glossary-entries/_README.md

File: site/en/_partials/privacy-sandbox/glossary-entries/_README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Using glossary partials
+
+These partials each contain one glossary term. If you use any and need a subhead for them, you'll need to include the subhead inline in your content before the include, like so
+
+    ## Ad exchange
+
+    {% Partial 'privacy-sandbox/glossary-entries/ad-exchange.njk' %}
```
Comment (heyawhite) [inline site/en/_partials/privacy-sandbox/glossary-entries/_README.md:7]: This is failing the linter, and should be made a fenced codeblock. https://www.markdownguide.org/extended-syntax/#fenced-code-blocks

````
```txt
## Ad exchange

{% Partial 'privacy-sandbox/glossary-entries/ad-exchange.njk' %}
```
````

**Review by heyawhite — APPROVED** [review]
Once the README linter error is addressed, LGTM and will merge.

File: site/en/_partials/privacy-sandbox/glossary-entries/_README.md (hunk 1 above)
Comment (nancymic2) [reply site/en/_partials/privacy-sandbox/glossary-entries/_README.md]: Ha I originally had the three ticks run inline. My memory failed me! fixed

File: site/en/_partials/privacy-sandbox/glossary-entries/_README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Using glossary partials
```
[the remark this answers is not in the document]
Comment (nancymic2) [reply site/en/_partials/privacy-sandbox/glossary-entries/_README.md]: I did wonder why as four spaces also works for code blocks.

**Pull request merged**